test/cypress/pages/paying-for-college/school-search.js

```diff
@@ -0,0 +1,46 @@
+export class PfcSchoolSearch {
+
+  openConstants() {
+    cy.visit( '/paying-for-college2/understanding-your-financial-aid-offer/api/constants/' );
+  }
+
+  openNationalStats() {
+    cy.visit( '/paying-for-college2/understanding-your-financial-aid-offer/api/national-stats/' );
+  }
+
+  openOffer( name ) {
+    cy.visit( `/paying-for-college2/understanding-your-financial-aid-offer/offer/?${ name }` );
+  }
+
+  openProgram( name ) {
+    cy.visit( `/paying-for-college2/understanding-your-financial-aid-offer/api/program/${ name }/` );
+  }
+
+  openSchool( id ) {
+    cy.visit( `/paying-for-college2/understanding-your-financial-aid-offer/api/school/${ id }.json` );
+  }
+
+  school( term ) {
+    cy.visit( `/paying-for-college2/understanding-your-financial-aid-offer/api/search-schools.json?q=${ term }` );
+  }
```

Comment:
Is there a plan to use these currently-unused API visit methods in the near future? If not, let's remove them.

Reply:
I was not sure if there were any existing paying-for-college API tests. If you know that the API is tested by unit tests then these API visit methods can be removed.